osu.Game/Overlays/FirstRunSetup/ScreenImportFromStable.cs

```diff
@@ -132,12 +132,13 @@ private void updateStablePath()
 
                 copyInformation.AddLink("Learn more about how \"hard links\" work", LinkAction.OpenWiki, @"Client/Release_stream/Lazer/File_storage#via-hard-links");
             }
-            else if (RuntimeInfo.OS != RuntimeInfo.Platform.Windows)
+            else if (RuntimeInfo.OS != RuntimeInfo.Platform.Windows && RuntimeInfo.OS != RuntimeInfo.Platform.Linux)
```

Comment:
@peppy I think this is going to need updating to `IsDesktop` too? Would entail trying to migrate across filesystems.